.github/workflows/ci.yml

```diff
@@ -23,12 +23,32 @@ jobs:
             ${{ runner.os }}-maven-
       - name: Install dependencies
         run: make install
+      - name: Set up JDK 19 for compilation
```

Comment:
Would prefer to keep these description lines static, we'll forget to update these transient numbers

```suggestion
      - name: Set up JDK for compilation
```